src/DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp

```diff
@@ -195,25 +211,25 @@ struct DetAndInverseImpl<Symmetry<2, 1>, Index0, Index1,
     const T& s10 = get<3, 2>(tensor);
     const T& s11 = get<3, 3>(tensor);
 
-    Tensor<T, Symmetry<2, 1>, inverse_indices<Index0, Index1>> inv{};
-    T& u00 = get<0, 0>(inv);
-    T& u01 = get<0, 1>(inv);
-    T& u10 = get<1, 0>(inv);
-    T& u11 = get<1, 1>(inv);
-    T& v00 = get<0, 2>(inv);
-    T& v01 = get<0, 3>(inv);
-    T& v10 = get<1, 2>(inv);
-    T& v11 = get<1, 3>(inv);
-    T& w00 = get<2, 0>(inv);
-    T& w01 = get<2, 1>(inv);
-    T& w10 = get<3, 0>(inv);
-    T& w11 = get<3, 1>(inv);
-    T& x00 = get<2, 2>(inv);
-    T& x01 = get<2, 3>(inv);
-    T& x10 = get<3, 2>(inv);
-    T& x11 = get<3, 3>(inv);
-
-    const T det_p = p00 * p11 - p01 * p10;
+    T& u00 = get<0, 0>(*inv);
+    T& u01 = get<0, 1>(*inv);
+    T& u10 = get<1, 0>(*inv);
+    T& u11 = get<1, 1>(*inv);
+    T& v00 = get<0, 2>(*inv);
+    T& v01 = get<0, 3>(*inv);
+    T& v10 = get<1, 2>(*inv);
+    T& v11 = get<1, 3>(*inv);
+    T& w00 = get<2, 0>(*inv);
+    T& w01 = get<2, 1>(*inv);
+    T& w10 = get<3, 0>(*inv);
+    T& w11 = get<3, 1>(*inv);
+    T& x00 = get<2, 2>(*inv);
+    T& x01 = get<2, 3>(*inv);
+    T& x10 = get<3, 2>(*inv);
+    T& x11 = get<3, 3>(*inv);
+
+    get(*det) = p00 * p11 - p01 * p10;
```

Comment:
Add comment above this line to clarify det is holding a temporary value. E.g., "// Temporarily store det(P) in `det`"